appveyor.yml

```diff
@@ -0,0 +1,30 @@
+image: ubuntu
+
+version: 3.6.0-{build}
+
+branches:
```

Comment:
Will need to remove that branch check as we'll want it to run all branches

Reply:
Then it will package on all branches.

Reply:
nope `disable_publish_on_pr: true` :)

Reply:
what will that do for feature branches pre-pr?  wouldn't they result in a build?

Reply:
yeah so appveyor will build and test but not pack which is what we want for PRs